client/notebooks/Demo_Client_Notebook_OpenDP_Polars.ipynb

```diff
@@ -0,0 +1,584 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "0fdebac9-57fc-4410-878b-5a77425af634",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Server error status 500: Internal Server Error\n"
+     ]
+    }
+   ],
+   "source": [
+    "income_metadata = client.get_dataset_metadata()\n",
+    "income_metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a3c899d",
+   "metadata": {},
+   "source": [
+    "### Get a dummy dataset\n",
+    "\n",
+    "Now, that she has seen and understood the metadata, she wants to get an even better understanding of the dataset (but is still not able to see it). A solution to have an idea of what the dataset looks like it to create a dummy dataset. \n",
+    "\n",
+    "Based on the public metadata of the dataset, a random dataframe can be created created. By default, there will be 100 rows and the seed is set to 42 to ensure reproducibility, but these 2 variables can be changed to obtain different dummy datasets.\n",
+    "Getting a dummy dataset does not affect the budget as there is no differential privacy here. It is not a synthetic dataset and all that could be learn here is already present in the public metadata (it is created randomly on the fly based on the metadata).\n",
+    "\n",
+    "Dr. FSO first create a dummy dataset with 200 rows and chooses a seed of 0."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "01f4365a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "NB_ROWS = 200\n",
+    "SEED = 0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "3f553b29",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Server error status 500: Internal Server Error\n"
+     ]
+    },
+    {
+     "ename": "AttributeError",
+     "evalue": "'NoneType' object has no attribute 'collect'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[6], line 3\u001b[0m\n\u001b[1;32m      1\u001b[0m dummy_lf \u001b[38;5;241m=\u001b[39m client\u001b[38;5;241m.\u001b[39mget_dummy_lf(nb_rows\u001b[38;5;241m=\u001b[39mNB_ROWS, seed \u001b[38;5;241m=\u001b[39m SEED)\n\u001b[0;32m----> 3\u001b[0m \u001b[43mdummy_lf\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mcollect\u001b[49m()\n",
+      "\u001b[0;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'collect'"
```

Comment:
maybe re-run to remove error and make sure it is ok ?

Reply:
ah oups! nice catch i pushed the one with the wrong outputs!